docs/home/contribute/localization.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Localizing Kubernetes Documentation
+---
+
+We're happy to add localizations (l10n) of Kubernetes documentation to the website!
+
+Localizations must meet the following requirements for _workflow_ (how to localize) and _output_ (what to localize).
+
+## Workflow  
+
+All l10n work must be stored and tracked within the [Kubernetes organization](https://github.com/kubernetes).
+
+#### Repository
+
+A l10n team will have a repository specifically dedicated to its work, for example: [kubernetes/kubernetes-docs/cn](https://github.com/kubernetes/kubernetes-docs-cn).
+
+**Note:** To open a l10n repository, [contact the SIG docs lead](https://kubernetes.slack.com/messages/C1J0BPD2M) on Slack for assistance.
+{: .note}
+
+#### Project
+
+Teams must track their overall progress with a [GitHub project](https://help.github.com/articles/creating-a-project-board/).
+
+Projects must include columns for:
+- pending work
+- work in progress
+- completed work
+
+#### Team function
+
+L10n teams must provide a single point of contact: the name and contact information of a person who can respond to or redirect questions or concerns.
+
+L10n teams must provide their own repository maintainers.
+
```

Comment:
FWIIW: Our cn i10n uses a lot more detailed tracking internally (via a Google spreadsheet): for each PR, we track its status, author, reviewer, etc. I guess these are considered as each team's internal implementation details and don't need to be standardized